batchflow/named_expr.py

```diff
@@ -639,19 +618,29 @@ class R(NamedExpression):
         R('poisson', lam=5.5, seed=42, size=3)
         R(['metro', 'taxi', 'bike'], p=[.6, .1, .3], size=10)
     """
-    def __init__(self, name, *args, state=None, seed=None, size=None, **kwargs):
+    def __init__(self, name, *args, seed=None, size=None, **kwargs):
         super().__init__(name)
-        if isinstance(state, np.random.RandomState):
-            self.random_state = state
-        else:
-            self.random_state = np.random.RandomState(seed)
         self.args = args
         self.kwargs = kwargs
+        self.random = make_rng(seed)
```

Comment:
You need it only if `self.default_random` is False

Reply:
Not only that. If pipeline is not specified, as well.